moveit_core/robot_state/src/robot_state.cpp

```diff
@@ -446,9 +446,8 @@ void RobotState::setJointEfforts(const JointModel* joint, const double* effort)
 void RobotState::setJointGroupPositions(const JointModelGroup* group, const double* gstate)
 {
   if (group == nullptr)
-  {
     return;
-  }
+
```

Comment:
Even if the parentheses were removed, it was confirmed that the code had the same execution result.